tools/vscode/src/components/webview.ts

```diff
@@ -24,22 +18,11 @@ export class InspectWebviewManager<T extends InspectWebview<S>, S> {
     private webviewType_: new (
       context: ExtensionContext,
       state: S,
-      webviewPanel: WebviewPanel
-    ) => T
+      webviewPanel: HostWebviewPanel
+    ) => T,
+    private host_: ExtensionHost
   ) {
     this.extensionUri_ = context.extensionUri;
-
```

Comment:
What happens now to Inspect View in VS Code when you do a full reload? Does it restore itself (seems like it might be able to, even if it just goes back to the default view). I'm concerned that by removing this code we end up with a dead Inspect View panel, but LMK what you observe here.

Reply:
I observe that it doesn't orphan a window, but I agree with you that this seems sketchy. I am reverting that change.